scripts/training/bulletproof_training_cell.py

```
@@ -4,22 +4,19 @@
-import pickle
 import torch
 import torch.nn as nn
-import numpy as np
 import pandas as pd
 from datasets import load_dataset
 from torch.utils.data import Dataset, DataLoader
 from sklearn.model_selection import train_test_split
 from sklearn.metrics import f1_score, accuracy_score
 from sklearn.preprocessing import LabelEncoder
 from transformers import AutoModel, AutoTokenizer
+from pathlib import Path
```

Ruff (0.12.2)

Module level import not at top of file

(E402)